agent/bench-scripts/test-bin/mock-postprocess/user-benchmark-wrapper

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Instead of mocking out user-benchmark-wrapper, could you have the test setup provide the proper input to test how user-benchmark-wrapper works?  I'd like to see four tests:

 1. Test a normal `.json` file with a duration
 2. Test a normal `.json` file without a duration
 3. Test a non-existent `.json` file
 4. Test a badly formatted `.json` file

What do you think?